specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/vaults.json

```diff
@@ -0,0 +1,2160 @@
+          "type": "string"
+        }
+      }
+    },
+    "ClientDiscoveryForLogSpecification": {
+      "description": "Class to represent shoebox log specification in json client discovery.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the log.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Localized display name",
+          "type": "string"
+        },
+        "blobDuration": {
+          "description": "Blobs created in customer storage account per hour",
+          "type": "string"
+        }
+      }
+    },
+    "ClientDiscoveryForProperties": {
+      "description": "Class to represent shoebox properties in json client discovery.",
+      "type": "object",
+      "properties": {
+        "serviceSpecification": {
+          "$ref": "#/definitions/ClientDiscoveryForServiceSpecification",
+          "description": "Operation properties."
+        }
+      }
+    },
+    "ClientDiscoveryForServiceSpecification": {
+      "description": "Class to represent shoebox service specification in json client discovery.",
+      "type": "object",
+      "properties": {
+        "logSpecifications": {
+          "description": "List of log specifications of this operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientDiscoveryForLogSpecification"
+          }
+        }
+      }
+    },
+    "ClientDiscoveryResponse": {
+      "description": "Operations List response which contains list of available APIs.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of available operations.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientDiscoveryValueForSingleApi"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next chunk of the response",
+          "type": "string"
+        }
+      }
+    },
+    "ClientDiscoveryValueForSingleApi": {
+      "description": "Available operation details.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Operation.",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/ClientDiscoveryDisplay",
+          "description": "Contains the localized display information for this particular operation"
+        },
+        "origin": {
+          "description": "The intended executor of the operation;governs the display of the operation in the RBAC UX and the audit logs UX",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/ClientDiscoveryForProperties",
+          "description": "ShoeBox properties for the given operation."
+        }
+      }
+    },
+    "CrossSubscriptionRestoreSettings": {
+      "description": "Settings for Cross Subscription Restore Settings",
```

Comment:
>Settings for Cross Subscription Restore Settings [](http://example.com/codeflow?start=22&length=48)

Can you provide more details on what these settings would do

Reply:
This setting is related to one of the capabilities in Azure Backup Vault called cross subscription restore.
We provide user an option to choose whether that want it to be Enabled/Disable/PermanentlyDisabled